diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47d5bbd..0ed40a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,8 @@ set(LIBRARY_VERSION 1.0.${CMAKE_PROJECT_VERSION})
 
 option(BUILD_SHARED_LIBS "Build dynamic library" ON)
 
+set(CMAKE_INSTALL_RPATH $ORIGIN)
+
 add_library(PortTime src/porttime/porttime.c)
 target_include_directories(PortTime PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
